getting_started/step_by_step/your_first_game.rst

```diff
@@ -666,8 +666,11 @@ This completes the `Mob` scene.
 Main scene
 ----------
 
-Now it's time to bring it all together. Create a new scene and add a
-:ref:`Node <class_Node>` named ``Main``. Click the "Instance" button and select your
+Now it's time to bring it all together. Create a new scene, click on
+the other node button and create a
+:ref:`Node <class_Node>` node as the root of the scene, it has a
+white circle as an icon and should be selected by default. Name
```

Comment:
The "," should be the start of a new sentence.

> `Node` is the first node type in the list and has a white circle icon.

Note: it probably *won't* be selected by default if the reader has done any searching of the node list previously. I'd recommend removing that statement.